telco-examples/mgmt-cluster/airgap/eib/mgmt-cluster-airgap.yaml

```diff
@@ -85,31 +84,59 @@ kubernetes:
-    - name: registry.rancher.com/rancher/mirrored-longhornio-longhorn-engine:v1.7.2
-    - name: registry.rancher.com/rancher/mirrored-longhornio-longhorn-instance-manager:v1.7.2
-    - name: registry.rancher.com/rancher/mirrored-longhornio-longhorn-manager:v1.7.2
-    - name: registry.rancher.com/rancher/mirrored-longhornio-longhorn-share-manager:v1.7.2
-    - name: registry.rancher.com/rancher/mirrored-longhornio-longhorn-ui:v1.7.2
-    - name: registry.rancher.com/rancher/system-upgrade-controller:v0.14.2
-    - name: registry.suse.com/rancher/seedimage-builder:1.6.4
+    - name: quay.io/jetstack/cert-manager-cainjector:v1.14.2
+    - name: quay.io/jetstack/cert-manager-controller:v1.14.2
+    - name: quay.io/jetstack/cert-manager-webhook:v1.14.2
+    - name: rancher/hardened-cluster-autoscaler:v1.9.0-build20241126
+    - name: rancher/hardened-cni-plugins:v1.6.2-build20250124
+    - name: rancher/hardened-cni-plugins:v1.6.2-build20250306
+    - name: rancher/hardened-coredns:v1.12.0-build20241126
+    - name: rancher/hardened-k8s-metrics-server:v0.7.2-build20250110
+    - name: rancher/hardened-multus-cni:v4.1.4-build20250108
+    - name: rancher/klipper-helm:v0.9.4-build20250113
+    - name: rancher/mirrored-cilium-cilium:v1.17.1
+    - name: rancher/mirrored-cilium-operator-generic:v1.17.1
+    - name: rancher/mirrored-longhornio-csi-attacher:v4.8.1
+    - name: rancher/mirrored-longhornio-csi-node-driver-registrar:v2.13.0
+    - name: rancher/mirrored-longhornio-csi-provisioner:v5.2.0
+    - name: rancher/mirrored-longhornio-csi-resizer:v1.13.2
+    - name: rancher/mirrored-longhornio-csi-snapshotter:v8.2.0
+    - name: rancher/mirrored-longhornio-livenessprobe:v2.15.0
+    - name: rancher/mirrored-longhornio-longhorn-engine:v1.8.1
+    - name: rancher/mirrored-longhornio-longhorn-instance-manager:v1.8.1
+    - name: rancher/mirrored-longhornio-longhorn-manager:v1.8.1
+    - name: rancher/mirrored-longhornio-longhorn-share-manager:v1.8.1
+    - name: rancher/mirrored-longhornio-longhorn-ui:v1.8.1
+    - name: rancher/mirrored-sig-storage-snapshot-controller:v8.2.0
+    - name: rancher/neuvector-compliance-config:1.0.4
+    - name: rancher/neuvector-controller:5.4.3
+    - name: rancher/neuvector-enforcer:5.4.3
+    - name: rancher/nginx-ingress-controller:v1.12.1-hardened1
+    - name: registry.opensuse.org/isv/suse/edge/3.3/images/3.3/endpoint-copier-operator:0.2.0
+    - name: registry.opensuse.org/isv/suse/edge/3.3/images/3.3/metallb-controller:v0.14.9
+    - name: registry.opensuse.org/isv/suse/edge/3.3/images/3.3/metallb-speaker:v0.14.9
+    - name: registry.opensuse.org/isv/suse/edge/factory/images/baremetal-operator:0.9.1
+    - name: registry.opensuse.org/isv/suse/edge/factory/images/ironic-ipa-downloader:3.0.6
+    - name: registry.opensuse.org/isv/suse/edge/factory/images/ironic:26.1.2.4
+    - name: registry.opensuse.org/isv/suse/edge/factory/images/upgrade-controller:0.1.1
```

Comment:
This is using the pre-release OBS builds so will need updating to match the final release images - although I think EIB can detect the helm chart related images anyway so perhaps we can remove these?

Reply:
we can, but I'd say to do an upgrade after having the artefacts created to have the full list. I don't want to reduce the list to avoid another issues

Reply:
Push forced with the minimum list to be tested